src/dialects/abstract/query-interface-typescript.ts

```diff
@@ -12,16 +14,99 @@ export class AbstractQueryInterfaceTypeScript {
   }
 
   /**
-   * Creates a new database schema.
+   * Create a new database schema.
    *
    * **Note:** this is a schema in the [postgres sense of the word](http://www.postgresql.org/docs/9.1/static/ddl-schemas.html),
-   * not a database table. In mysql and sqlite, this command will do nothing.
+   * not a database table. In mysql and mariadb, this command will instead create a database.
```

Comment:
I would phrase it like this, since the mention of this not creating a database table doesn't really make sense

```
   * **Note:** this is a schema in the [postgres sense of the word](http://www.postgresql.org/docs/9.1/static/ddl-schemas.html): A collection of tables. 
   * In mysql and mariadb, this command will create what they call a database.
```

Also valid for the other places where this comment has been added